pkgs/by-name/jo/jocalsend/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchFromGitea,
+  rustPlatform,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "jocalsend";
+  version = "1.6.1803";
+
+  src = fetchFromGitea {
+    domain = "git.kittencollective.com";
+    owner = "nebkor";
+    repo = "joecalsend";
+    rev = finalAttrs.version;
```

We can use tag for this fetcher

```suggestion
    tag = finalAttrs.version;
```